cmd/ci-operator-config-mirror/main.go

```diff
@@ -51,6 +56,45 @@ func (o *options) validate() error {
 	return o.WhitelistOptions.Validate()
 }
 
+type dataWithInfoByRepo map[string][]config.DataWithInfo
```

Comment:
Can we call the type `configsByRepo`? It's simpler and carries better meaning, IMO.